src/main/java/service/operation/ReturnOperation.java

```diff
@@ -0,0 +1,22 @@
+package service.operation;
+
+import data.db.Storage;
+import java.util.Map;
+import model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+
+    @Override
+    public void operation(FruitTransaction.Operation operation, FruitTransaction fruitTransaction) {
+        if (fruitTransaction.getQuantity() <= 0) {
+            return;
+        }
+        for (Map.Entry<String, Integer> fruit : Storage.fruitsStorage.entrySet()) {
+            if (fruit.getKey().equals(fruitTransaction.getFruit())) {
+                fruit.setValue(fruit.getValue() + fruitTransaction.getQuantity());
+                return;
+            }
+        }
+        Storage.fruitsStorage.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
```

Comment:
Using a loop to find a fruit in the storage is not efficient. Consider using a data structure that allows faster access, such as a Map, and use its methods to check and update the fruit's quantity.